asv_bench/benchmarks/groupby.py

```diff
@@ -243,7 +208,7 @@ def time_multi_count(self, df):
         df.groupby(['key1', 'key2']).count()
 
 
-class CountInt(object):
+class CountMultiInt(object):
```

Comment:
`int` tests for `count` / `nunique` are already covered in `GroupByMethods` though the differentiator here is that we are testing the use of multiple fields in the grouping. It may make sense to just do that natively in `GroupByMethods` but I didn't want to rip up too many things as of yet